website/content/docs/operations/health.mdx

```diff
@@ -0,0 +1,100 @@
+---
+layout: docs
+page_title: Boundary Health Endpoints
+description: |-
+  Verify the Boundary controller server is up and able to receive requests
+---
+
+## Boundary Health Endpoints
+
+Boundary provides health monitoring through the `/health` path using a listener
+with the `"ops"` purpose. By default a listener with that purpose runs on port
+`9203`. See the [example configuration](#example-configuration) section for an
+example listener stanza in a `config.hcl` file.
+
+### Requirements
+
+For the controller health endpoint to be enabled, any Boundary instance must be
+started with a controller. That is, a `controller` block and a `purpose = "api"`
+listener must be defined in Boundary's configuration file. Additionally, an
+`purpose = "ops"` listener must also be defined in Boundary's configuration
+file. Under these conditions the ops server (which hosts the controller health
+api) will be exposed.
+
+### Shutdown Grace Period
+
+Optionally, when the controller health endpoint is enabled Boundary can be
```

Comment:
I'd like to suggest a change with this sentence to break it up into multiple sentences so that it reads well.  Please feel free to nix my suggestion if it doesn't make sense or I've somehow changed the intent of the sentence. First off, typically it's best to avoid starting a sentence off with 'Optionally.' 

Here's my suggestion:
When the controller health endpoint is enabled, Boundary may be configured to change the controller health response to a`403 Server Unavailable`. The controller health response will only change upon receiving a shutdown signal. An amount of time may be configured before the shutdown process begins.